### PR TITLE
mobile-friendly-ish styles

### PR DESCRIPTION
**Pull request by stepansnigirev**
Getting started with https://github.com/cryptoadvance/specter-desktop/issues/26
For now only hiding sidebar if the screen is small.
Navbar appears when hovered, for touchscreens if clicked on the arrow.

ezgif-7-12ad306c0c46

### CODE REVIEW

**Comment by ben-kaufman** [comment]
That's very cool! great start for mobile. Tested on desktop and mobile and looks nice.
I have encountered 2 issues with that though.
First is when you have log in required, the logout button covers the top of the sidebar. (see image)

pr112


Second is that on a mobile device, it is nearly impossible to click the sidebar to expand it. I think it'd be better to increase the width a bit or maybe change it to some larger button in case the device is mobile.

**Comment by stepansnigirev** [comment]
I plan to continue working on this PR, so I will mark it as WIP.

@ben-kaufman thanks for testing it, I will pay attention to the login button.

Next - scale for mobile:
`<meta name="viewport" content="width=device-width, initial-scale=1.0">`
 and making sure it fits on the screen

**Comment by stepansnigirev** [comment]
Made some changes. It still looks bad on mobile in portrait mode, but in landscape looks reasonable.

@ben-kaufman can you test and give me some feedback?

**Comment by ben-kaufman** [comment]
@stepansnigirev works great!
The new menu button for touchscreens is really nice :)
I tried on both a phone and a tablet. On the tablet it works great on both portrait and landscape. On a phone the landscape is fine, but I guess that for real support of such small screens we'll need many more changes, but for now this is very good.

File: src/cryptoadvance/specter/static/styles.css [hunk 1]
```diff
@@ -423,7 +446,7 @@ input:hover{
 }
 .note{
     font-size: 0.7em;
-    width: 400px;
+    max-width: 400px;
```
Comment (ben-kaufman) [inline src/cryptoadvance/specter/static/styles.css:449]: Seems like this causes the note on the send screen (showing the available coins and sat/btc conversion to appear a bit strange.

Comment (stepansnigirev) [reply src/cryptoadvance/specter/static/styles.css:449]: hopefully better now, added a few tweak.
In the `.card` element note will be left-aligned.

**Comment by stepansnigirev** [comment]
Yes, for mobile in portrait mode we would need many changes.
I would like to keep this PR for initial mobile & tablet support and get to the phone version a bit later.

**Comment by ben-kaufman** [comment]
Looks good now on Chrome, but suddenly very bad on Safari (for example, the menu is visible on desktop, and in the Addresses tab the History/UTXO is strangely stretched).

**Comment by ben-kaufman** [comment]
Hmmm but I see that without your changes Chrome looks quite bad. So it's better that we support Chrome properly but best if we can make it look good on both.

**Comment by stepansnigirev** [comment]
can you make screenshots? for me in safary looks ok...
Maybe safary cached something? You need to cmd+option+e to clear the cache.

Screenshot 2020-05-12 at 17 21 13

**Comment by ben-kaufman** [comment]
Yes, apparently it was caching issue so nevermind. Works great!

**Pull request merged**